File: mini-commerce/src/main/java/me/hero/minicommerce/item/application/port/in/ModifyItemUseCase.java
package me.hero.minicommerce.item.application.port.in;

import me.hero.minicommerce.item.application.port.in.dto.ModifiedItemDto;
import me.hero.minicommerce.item.application.port.in.dto.ModifyItemDto;

public interface ModifyItemUseCase {
  ModifiedItemDto modifyItem(Long itemId, ModifyItemDto modifyItemDto);
}

File: mini-commerce/src/main/kotlin/me/hero/minicommerce/item/application/port/in/ModifyItemUseCase.kt
package me.hero.minicommerce.item.application.port.`in`

import me.hero.minicommerce.item.application.port.`in`.dto.ModifiedItemDto
import me.hero.minicommerce.item.application.port.`in`.dto.ModifyItemDto


interface ModifyItemUseCase {
    fun modifyItem(itemId: Long, modifyItemDto: ModifyItemDto): ModifiedItemDto
}